test/Fakes/FakeCredentialProvider/TestCredentialRequest.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Test.Fakes.FakeCredentialProvider
+{
+    public class TestCredentialRequest
```

Comment:
Please don't use the term Fake to describe test providers. Fake has other connotations and known framework names,